src/pages/PlayerProfile.tsx

```diff
@@ -102,9 +102,11 @@ export const PlayerProfile = (props : PlayerProfileProps) => {
 
     function handlePlayerDetailsSubmitButton() {
         if (imagePreview !== '/broken-image.jpg') {
+            localStorage.clear();
```

Comment:
You can just call it once
```suggestion
    function handlePlayerDetailsSubmitButton() {
        localStorage.clear();
        if (imagePreview !== '/broken-image.jpg') {
```